canopy_main.ml

```diff
@@ -41,29 +41,28 @@ module Main  (C: CONSOLE) (S: STACKV4) (RES: Resolver_lwt.S) (CON: Conduit_mirag
     let module Store = Canopy_store.Store(C)(Context)(Inflator) in
     let open Canopy_utils in
     let cache = ref (KeyMap.empty) in
-    let open Canopy_config in
-    let config = config () in
+    Store.pull console >>= fun () ->
+    Store.fill_cache cache >>= fun l ->
     let update_atom, atom =
-      Canopy_syndic.atom config Store.last_commit_date cache
+      Canopy_syndic.atom Store.last_commit_date cache
     in
     let store_ops = {
       Canopy_dispatch.subkeys = Store.get_subkeys ;
       value = Store.get_key ;
       update =
         (fun () ->
-           cache := KeyMap.empty;
```

Comment:
see https://github.com/Engil/Canopy/pull/64/commits/cd5aff8e1936f4fa5b01f93947b43e9b8b6dd52b (part of https://github.com/Engil/Canopy/pull/64)